### PR TITLE
README.md cleanup

### PR DESCRIPTION
**Pull request by rajhraval**
Cleaned up the Quote Format and converted the style in Code Format

### CODE REVIEW

**Review by ygit — DISMISSED** [review]
Looks good to merge 🎉

File: ChatkitServer/README.md [hunk 1]
```diff
@@ -4,13 +4,15 @@
...
+* Make sure you have installed Vapor 3.1.0 or higher version. Installation steps on  [Vapor-Codes](https://docs.vapor.codes/3.0/install/macos)
 
-* Once installed, follow with these commands in the terminal
+* Once installed, follow with these commands in the terminal:
 
-> cd ChatkitServer
-> vapor build
-> vapor run
+    ```
+    cd ChatkitServer
```
Comment (lsampras) [inline ChatkitServer/README.md]:
```suggestion
    $ cd ChatkitServer
```

File: ChatkitServer/README.md [hunk 2]
```diff
@@ -4,13 +4,15 @@
...
+    vapor build
```
Comment (lsampras) [inline ChatkitServer/README.md]:
```suggestion
    $ vapor build
```

File: ChatkitServer/README.md [hunk 3]
```diff
@@ -4,13 +4,15 @@
...
+    vapor run
```
Comment (lsampras) [inline ChatkitServer/README.md]:
```suggestion
    $ vapor run
```

**Review by ygit — DISMISSED** [review]

**Comment by ygit** [comment]
Looks good to merge 🎉

**Review by lsampras — APPROVED** [review]
LGTM

**Review by shwetsolanki — APPROVED** [review]

**Review by ygit — APPROVED** [review]
Looks good to merge 🎉

**Pull request merged**